pages/sdk/onramp-kit/guides/stripe.mdx

```diff
@@ -1,119 +1,152 @@
-import { Steps } from 'nextra/components'
+import { Steps, Callout } from 'nextra/components'
+import Image from 'next/image'
 
 # Integration with Stripe
 
```

Comment:
Can we add some information about what is possible through this Integration like you did for Monerium here? https://github.com/safe-global/safe-docs/pull/553/files#diff-efc102eaaa40c277c1a243efd02c54933ba1f39eb22ab662d1cac4add1b6f276R7